packages/core/src/components/tabs/tab.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
```

Comment:
🤔 original date?

Reply:
I'm down to leave the original date.

Reply:
I like to use `2015-present` since it's never out of date